changelog/b6227.dd

```diff
@@ -1,24 +1,21 @@
-Comparison of values belonging to different enums is deprecated.
+Comparison that takes into account the enumerated type improves code correctness by
+disallowing comparison of unrelated enumerated types. 
```

Comment:
Yes it can't be and we should trigger a hard failure so that people, who don't read the README, are warned by the CI.

Also it seems that reviewers don't look at the changelog preview either as there's currently a missing parentheses:

http://dlang.org/changelog/2.075.0_pre.html

The Ddoc macro wouldn't be visible otherwise...